### PR TITLE
Feature: update README.md

### PR DESCRIPTION
**Pull request by kborisov00**
## Notes

- Removed completely the "Usage" section.

  - Instead, I've added a legend text which recommends to users to get familiar with the Turborepo documentation first. Everything about how to work with monorepos is already very well documented there and I think it's better to read their documentation. This way we'll keep the README clean and consistent with the other projects and we won't repeat ourselves with what's already been documented.

#### Todo

- [x] Change the logo inside `README.md`

## Checklist

- [ ] All new changes were tested on various devices
- [ ] All change-related components were tested for regression 
- [ ] All new changes were tested with missing or incorrect data

### CODE REVIEW

**Review by SMerdzhanov — APPROVED** [review]
Hey @kborisov00, I've updated the README.md.

Most areas should now be done. There are a few `<!-- TODO -->` comments left and such that reflect our last review of the changes.

Please take your time to update the README.md as agreed so we complete this task.

File: README.md [hunk 1]
```diff
@@ -1 +1,236 @@
...
+    <img alt="NextJS" src="https://camo.githubusercontent.com/92ec9eb7eeab7db4f5919e3205918918c42e6772562afb4112a2909c1aaaa875/68747470733a2f2f6173736574732e76657263656c2e636f6d2f696d6167652f75706c6f61642f76313630373535343338352f7265706f7369746f726965732f6e6578742d6a732f6e6578742d6c6f676f2e706e67" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  Monorepo Starter Essentials
+</h1>
+
+Monorepo project starter extended with tools to help you boost your development experience.
+
+> Help us improve by [submitting suggestions and bug reports](https://github.com/MK-IT/nextjs-starter-essentials/issues).\_
```
[the remark this answers is not in the document]
Comment (kborisov00) [reply README.md]: This was already fixed, not sure how you came across this change.

File: README.md [hunk 2]
```diff
@@ -1 +1,236 @@
-# Monorepo Starter Essentials
+<p align="center">
+  <a href="https://nextjs.org">
+    <img alt="NextJS" src="https://camo.githubusercontent.com/92ec9eb7eeab7db4f5919e3205918918c42e6772562afb4112a2909c1aaaa875/68747470733a2f2f6173736574732e76657263656c2e636f6d2f696d6167652f75706c6f61642f76313630373535343338352f7265706f7369746f726965732f6e6578742d6a732f6e6578742d6c6f676f2e706e67" width="60" />
```
[the remark this answers is not in the document]
Comment (kborisov00) [reply README.md]: This was already fixed, not sure how you came across this change.

File: README.md [hunk 3]
```diff
@@ -1 +1,236 @@
-# Monorepo Starter Essentials
+<p align="center">
+  <a href="https://nextjs.org">
```
[the remark this answers is not in the document]
Comment (kborisov00) [reply README.md:2]: This was already fixed, not sure how you came across this change.

File: README.md [hunk 4]
```diff
@@ -1 +1,236 @@
...
+    <img alt="NextJS" src="https://camo.githubusercontent.com/92ec9eb7eeab7db4f5919e3205918918c42e6772562afb4112a2909c1aaaa875/68747470733a2f2f6173736574732e76657263656c2e636f6d2f696d6167652f75706c6f61642f76313630373535343338352f7265706f7369746f726965732f6e6578742d6a732f6e6578742d6c6f676f2e706e67" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  Monorepo Starter Essentials
+</h1>
+
+Monorepo project starter extended with tools to help you boost your development experience.
```
[the remark this answers is not in the document]
Comment (kborisov00) [reply README.md]: This was already fixed, not sure how you came across this change.

**Pull request merged**